config/fcos/v1_5_exp/translate.go

```diff
@@ -292,3 +302,51 @@ func translateBootDeviceLuks(from BootDeviceLuks, options common.TranslateOption
 	tm.AddTranslation(path.New("yaml"), path.New("json"))
 	return
 }
+
+func (c Config) processPackages(options common.TranslateOptions) (types.Config, report.Report, translate.TranslationSet) {
+	yamlPath := path.New("yaml", "inline")
```

Comment:
In struct translation functions, the path is relative to the root of the struct.  In desugaring functions, though, it's relative to the root of the config.  So this isn't correct; you need a full path here.

Reply:
I am not entirely sure I get which config. My latest commit I assumes is the config the user provides not the types.Config right?. So `yamlPath := path.New("yaml", "extensions")` ?

Reply:
Right, the root of the input (YAML, Butane) config.